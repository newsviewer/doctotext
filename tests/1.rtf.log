Using RTF parser.
[cmd: rtf (1)]
[cmd: ansi (0)]
[cmd: ansicpg (1252)]
Initializing converter for codepage 1252
Converter initialized.
[cmd: deff (0)]
[cmd: deftab (709)]
[cmd: fonttbl (0)]
[cmd: f (0)]
[cmd: fnil (0)]
[cmd: fcharset (0)]
Setting win charset 0 for font number 0
[cmd: colortbl (0)]
[cmd: red (0)]
[cmd: green (0)]
[cmd: blue (255)]
[cmd: red (0)]
[cmd: green (0)]
[cmd: blue (128)]
[cmd: * (42)]
[cmd: generator (0)]
[cmd: viewkind (4)]
[cmd: uc (1)]
[cmd: pard (0)]
[cmd: lang (1033)]
[cmd: f (0)]
Font number 0 referenced. Setting converter for encoding CP1250
[cmd: fs (24)]
[cmd: field (0)]
[cmd: * (42)]
[cmd: fldinst (0)]
[cmd: fldrslt (0)]
[cmd: cf (2)]
[cmd: ul (0)]
[cmd: cf (0)]
[cmd: ulnone (0)]
[cmd: f (0)]
Font number 0 referenced. Setting converter for encoding CP1250
[cmd: fs (24)]
[cmd: par (0)]
