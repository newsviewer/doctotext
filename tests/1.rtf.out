<HYPERLINK "http://www.silvercoders.com/" >hyperlink test

